meinberlin/apps/projects/templates/meinberlin_projects/project_detail.html

```diff
@@ -74,66 +74,61 @@ <h1 class="title">{{ project.name }}</h1>
             </div>
         </div>
     </section>
-
-    <!-- old styling -->
-    <div class="tablist tablist--left">
-        <div class="container">
-            <div class="tablist--right">
-                {% has_perm 'a4projects.change_project' request.user view.project as user_may_change %}
-                {% if user_may_change %}
-                    <a
-                        href="{% url 'a4dashboard:project-edit' project_slug=project.slug %}"
-                        class="btn btn--light btn--small"
-                        data-embed-target="external">
-                        {% translate 'Edit' %}
-                    </a>
-                {% endif %}
-            </div>
-
-            <nav class="nav" role="tablist">
-                <a
-                    id="tab-project-{{ project.pk }}-information"
-                    class="tab"
-                    data-bs-toggle="tab"
-                    href="#tabpanel-project-{{ project.pk }}-information"
-                    role="tab"
-                    aria-controls="tabpanel-project-{{ project.pk }}-information"
-                    aria-expanded="false">
-                    {% translate 'About the Project' %}
-                </a>
-                <a
-                    id="tab-project-{{ project.pk }}-participation"
-                    class="tab active"
-                    data-bs-toggle="tab"
-                    href="#tabpanel-project-{{ project.pk }}-participation"
-                    role="tab"
-                    aria-controls="tabpanel-project-{{ project.pk }}-participation"
-                    aria-expanded="true">
-                    {% translate 'Participation' %}
+    <div class="js-tabs">
+        <div class="row row--reverse-palm">
+            {% has_perm 'a4projects.change_project' request.user view.project as user_may_change %}
+            {% if user_may_change %}
+            <div class="span4 align--right">
+                <a class="link--edit" href="{% url 'a4dashboard:project-edit' project_slug=project.slug %}">
+                    {% translate 'Edit' %}
                 </a>
-                <a
-                    id="tab-project-{{ project.pk }}-result"
-                    class="tab"
-                    data-bs-toggle="tab"
-                    href="#tabpanel-project-{{ project.pk }}-result"
-                    role="tab"
-                    aria-controls="tabpanel-project-{{ project.pk }}-result"
-                    aria-expanded="false">
-                    {% translate 'Result' %}
-                </a>
-            </nav>
+            </div>
+            {% endif %}
+            <div class="span8">
+                <div role="tablist"
+                    aria-label="{% translate 'Project Details Navigation' %}">
```

Comment:
@hom3mad3 i read up this and according to mozilla we should have an aria-label for tablists if they don't have a visible heading and this is the best I could come up with?! is it descriptive enough do you think?